v2.0/ping/singularity-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "2.0"
+instances:
+  alp1:
+    build:
+      context: ./alp1
+      options:
+        - fakeroot
+    ports:
+      - "1025:1025"
+    start:
+      options:
+       - fakeroot
+    exec:
+      options: 
+        - "env-file=myvars.env"
+      command: printenv SUPERHERO
+  alp2:
+    build:
+      context: ./alp2
+      options:
+        - fakeroot
+    ports:
+      - "1026:1026"
+    start:
+      options:
+       - fakeroot
+    run: []
+    depends_on:
+      - alp1
```

Comment:
Might adding an example that illustrates the full set of new features help? Specifically run: options: and run: args:

Reply:
I don't have a good dummy example for this, because (as you pointed out) if the run command doesn't continue it will leave everything hanging. So I'm going to fight back a bit on this one and say we should wait until there is a real use case to exemplify, and the example with start and exec should be enough to get the user started.